lms/djangoapps/courseware/tests/test_block_render.py

```diff
@@ -925,7 +925,7 @@ def test_anonymous_user_not_be_graded(self, mock_score_signal):
         ('goto_position', False),  # does not set it
     )
     @ddt.unpack
-    @patch('lms.djangoapps.courseware.module_render.get_module_for_descriptor', wraps=get_module_for_descriptor)
+    @patch('lms.djangoapps.courseware.block_render.get_block_for_descriptor', wraps=get_block_for_descriptor)
     def test_will_recheck_access_handler_attribute(self, handler, will_recheck_access, mock_get_module):
```

Comment:
```suggestion
    def test_will_recheck_access_handler_attribute(self, handler, will_recheck_access, mock_get_block):
```

\+ its references in this method.